README.md

```diff
@@ -1,6 +1,6 @@
 [996.ICU](https://996.icu)
 =======
-### **Please note there are NO any official accounts, apps or merchandise except for the domain and this repository.**
+### **Please note that there does NOT exist any official account, app or merchandise except for the official domain and this repository.**
```

Comment:
Just say: `There are NO official accounts, apps`... One word change

Reply:
That would work as well.

Reply:
How about add "other"?

```suggestion
### **Please note that there does NOT exist any other official account, app or merchandise except for the official domain and this repository.**
```
```suggestion
### **Please note there are NO any other official accounts, apps or merchandise except for the domain and this repository.**
```